### ISSUE TITLE
bin/* creates conflicts

### ISSUE THREAD

**Issue by DannyBen** [waiting-for-dev/front_matter_parser#9]
The files in the `bin` folder should probably not be included in the final gem, as these seem to be intended for development only.

https://github.com/waiting-for-dev/front_matter_parser/blob/698034cdb33f19eed29fa0e445f7bb9592e51091/front_matter_parser.gemspec#L17

Since there are [other offending gems](https://github.com/wvanbergen/request-log-analyzer/blob/master/request-log-analyzer.gemspec#L53) that have binaries with the same name, when installing either of them, users get this error:

```
front_matter_parser's executable "console" conflicts with request-log-analyzer
Overwrite the executable? [yN]  n
ERROR:  Error installing eucalypt:
        "console" from front_matter_parser conflicts with installed executable from request-log-analyzer
```

**Comment by waiting-for-dev** [comment]
Thanks for reporting @DannyBen . Fixed and released in v0.2.1

**Comment by DannyBen** [comment]
Nice, thanks.
